ports/esp32/boards/UM_NANOS3/pins.csv

```diff
@@ -0,0 +1,10 @@
+I2C_SCL,GPIO8
+I2C_SDA,GPIO9
+SPI_SS,GPIO34
+SPI_MOSI,GPIO35
+SPI_SCK,GPIO36
+SPI_MISO,GPIO37
+RGB_DATA,GPIO41
+RGB_PWR,GPIO42
+UART0_X,GPIO43
```

Comment:
Shouldn't that be UART0_TX?

Reply:
Indeed it should, silly rushed copy & paste!
Fixed, thanks :)